index.html

```diff
@@ -1,52 +1,33 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
-<html lang="en">
+<!DOCTYPE html>
+<html>
 	<head>
-		<title>Demiurge Server Home Page</title>
+		<title>Demiurge - The build it as you go server</title>
+		<meta charset="utf-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0" />
+		<link rel="stylesheet" type="text/css" href="styles/style.css" />
+		<link rel="stylesheet" type="text/css" href="styles/layout.css" />
 
-		<meta name="description" content="" />
-		<meta http-equiv="Content-type" content="text/html; charset=UTF-8">
-		<meta http-equiv="Content-Style-Type" content="text/css">
-		<meta name="viewport" content="width=device-width, initial-scale=1">
-		<meta http-equiv="Content-Style-Type" content="text/css" />
-		<link rel="stylesheet" type="text/css" href="../css/navbar.css" />
-		<link rel="stylesheet" type="text/css" href="../css/basic.css" />
+		<!-- Responsive -->
+		<!-- <link rel="stylesheet" type="text/css" href="/desktop.css" media="screen and (min-width: 641px) and (min-height: 641px)" /> -->
+		<!-- <link rel="stylesheet" type="text/css" href="/mobile.css" media="screen and (max-width: 640px), screen and (max-height: 640px)" /> -->
+		<!-- <meta name="theme-color" content="#740000" /> -->
 	</head>
 
 	<body>
-		<div id="container">
-			<div id="header">
-			<!-- start of top section -->
-				<!--#include virtual="header.html" -->
-			<!-- end of top section -->
-			</div>
-			<div id="navcol">
-				<div id="navproper">
-				<!-- start of navigation side column -->
-					<!--#include virtual="navbar.html" -->
-				<!-- end of navigation side column -->
-				</div>
-			</div>
-			<div id="content">
-				<div id="contproper">
-					<!-- start of content column -->
-						<br>
-						<p>Todo : fill this space with automated scrape of announcements channel in discord.</p>
-						<br>
-					<!-- end of content column -->
+		<header>
+			<div><span>server status</span></div>
+			<div><span>player count</span></div>
+		</header>
+		<div>
+			<div>
+				<div>
+					<nav><span>NAV BAR</span></nav>
+					<main><span>new and announcements</span></main>
 				</div>
+				<footer><span>no one needs a footer</span></footer>
```

Comment:
foot space is for potential copyright notices if I use somebodies artwork, just fyi

I've merge  this to master as-is and the new "live" branch is whats perma on server so that we can continue to work off master and at times we can live check the results of changes by just branch switching on the server.